core/region.go

```diff
@@ -271,13 +271,13 @@ func (r *region) requestSpotPrices() error {
 		// spot market
 		price, err := strconv.ParseFloat(*priceInfo.SpotPrice, 64)
 		if err != nil {
-			logger.Println(r.name, "Instance type ", instType,
+			debug.Println(r.name, "Instance type ", instType,
 				"is not available on the spot market")
 			continue
 		}
 
 		if r.instanceTypeInformation[instType].pricing.spot == nil {
-			logger.Println(r.name, "Instance data missing for", instType, "in", az,
+			debug.Println(r.name, "Instance data missing for", instType, "in", az,
```

Comment:
Removes this wall of text from every invocation:
```
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for g4dn.2xlarge in us-east-1f skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for g4dn.xlarge in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for g4dn.16xlarge in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for g4dn.16xlarge in us-east-1f skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for g4dn.16xlarge in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for g4dn.16xlarge in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for g4dn.16xlarge in us-east-1c skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5n.large in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5n.large in us-east-1f skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5n.large in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5n.large in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5n.large in us-east-1c skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for a1.metal in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5dn.24xlarge in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5dn.24xlarge in us-east-1f skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5dn.24xlarge in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5dn.24xlarge in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5dn.24xlarge in us-east-1c skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for inf1.6xlarge in us-east-1f skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for inf1.6xlarge in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for inf1.6xlarge in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for c5n.metal in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for c5n.metal in us-east-1f skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for c5n.metal in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for c5n.metal in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for c5n.metal in us-east-1c skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5n.4xlarge in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5n.2xlarge in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for g4dn.12xlarge in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5n.12xlarge in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5n.2xlarge in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5n.4xlarge in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for c5d.24xlarge in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5n.2xlarge in us-east-1c skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5n.8xlarge in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5n.8xlarge in us-east-1f skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5n.8xlarge in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5n.8xlarge in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5n.8xlarge in us-east-1c skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5dn.xlarge in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.8xlarge in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.8xlarge in us-east-1f skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.8xlarge in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.8xlarge in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.8xlarge in us-east-1c skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5dn.xlarge in us-east-1c skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5n.12xlarge in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for g4dn.2xlarge in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5n.4xlarge in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for g4dn.xlarge in us-east-1c skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.xlarge in us-east-1f skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for c5d.12xlarge in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5dn.2xlarge in us-east-1f skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5dn.2xlarge in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5dn.2xlarge in us-east-1c skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.xlarge in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5n.xlarge in us-east-1c skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for g4dn.4xlarge in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for c5d.metal in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5n.8xlarge in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5n.8xlarge in us-east-1f skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5n.8xlarge in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5n.8xlarge in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5n.8xlarge in us-east-1c skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5n.xlarge in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5dn.2xlarge in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for inf1.24xlarge in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for inf1.24xlarge in us-east-1f skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for inf1.24xlarge in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for c5d.12xlarge in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for g4dn.2xlarge in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.xlarge in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for g4dn.2xlarge in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.2xlarge in us-east-1f skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.2xlarge in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for g4dn.xlarge in us-east-1f skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for g4dn.xlarge in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.2xlarge in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.large in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.large in us-east-1f skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.large in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.large in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.large in us-east-1c skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.4xlarge in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for g4dn.xlarge in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5dn.2xlarge in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for inf1.xlarge in us-east-1f skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for inf1.xlarge in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5n.large in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5n.large in us-east-1f skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5n.large in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5n.large in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5n.large in us-east-1c skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.xlarge in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5dn.8xlarge in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for c5d.24xlarge in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5dn.4xlarge in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5dn.4xlarge in us-east-1f skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5dn.4xlarge in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5dn.4xlarge in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5dn.4xlarge in us-east-1c skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5dn.large in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5dn.large in us-east-1f skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5dn.large in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5dn.large in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5dn.large in us-east-1c skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5n.12xlarge in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5n.12xlarge in us-east-1f skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5n.12xlarge in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5n.12xlarge in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5n.12xlarge in us-east-1c skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5dn.8xlarge in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5dn.8xlarge in us-east-1f skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5dn.8xlarge in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5dn.8xlarge in us-east-1c skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.12xlarge in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for a1.metal in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for a1.metal in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for g4dn.2xlarge in us-east-1c skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5n.xlarge in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for c5d.metal in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.12xlarge in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.xlarge in us-east-1c skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for c5d.12xlarge in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5n.xlarge in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.2xlarge in us-east-1c skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for g4dn.8xlarge in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5n.2xlarge in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.16xlarge in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.16xlarge in us-east-1f skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.16xlarge in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.16xlarge in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5n.2xlarge in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for c5d.24xlarge in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5n.4xlarge in us-east-1f skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5n.4xlarge in us-east-1c skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for g4dn.12xlarge in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for g4dn.12xlarge in us-east-1f skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for g4dn.12xlarge in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for g4dn.12xlarge in us-east-1c skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5dn.12xlarge in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5dn.12xlarge in us-east-1f skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5dn.12xlarge in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5dn.12xlarge in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5dn.12xlarge in us-east-1c skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5n.24xlarge in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5n.24xlarge in us-east-1f skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5n.24xlarge in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5n.24xlarge in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5n.24xlarge in us-east-1c skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.12xlarge in us-east-1f skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.12xlarge in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.12xlarge in us-east-1c skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5n.xlarge in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for g4dn.4xlarge in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.2xlarge in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5n.4xlarge in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5n.12xlarge in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5n.12xlarge in us-east-1f skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5n.12xlarge in us-east-1c skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5n.16xlarge in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5n.16xlarge in us-east-1f skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5n.16xlarge in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5n.16xlarge in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5n.16xlarge in us-east-1c skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.4xlarge in us-east-1b skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.4xlarge in us-east-1f skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.4xlarge in us-east-1a skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5dn.4xlarge in us-east-1c skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for r5n.xlarge in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for m5n.xlarge in us-east-1d skipping because this region is currently not supported
2020/01/25 17:32:07 region.go:280: us-east-1 Instance data missing for g4dn.4xlarge in us-east-1a skipping because this region is currently not supported

```